Service/Runner.php

```diff
@@ -458,6 +458,9 @@ private function runImages($jobId, $configId, $rowId, Component $component, Usag
                 $workingDirectory->moveOutputToInput();
             }
         }
+        if (($mode !== self::MODE_DEBUG) && $this->shouldStoreState($component->getId(), $configId)) {
```

Comment:
tady bych pro jistotu ještě přihodil ` && $image->isMail()`, v tom `shouldStoreState` nejsem schopnej vyklíčovat, jestli to je obsažený nebo ne.

Reply:
ono to tam neni, ale vyhodi se to na tom, ze procesor nema configuraci, pridam to tam, usetrime API call, ale nevim jestli to pujde otestovat, kdyz to funguje

Reply:
jasně, testovat se to bude asi blbě, jen jsem chtěl trochu zlepšit čitelnost pro další generace :-)